openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -214,6 +224,58 @@ This is an example of a Wallet Instance Attestation:
     * In the `input-descriptors` object, `id`, `name`, `purpose`, `group`, `format`, and `constraints` properties MUST be supported. In the `constraints` object, `limit_disclosure`, and `fields` properties MUST be supported. In the `fields` object, `path` and `filter` properties MUST be supported. A `path` MUST contain exactly one entry with a static path to a certain claim. A `filter` MUST only contain `type` elements of value `string` and `const` elements.
     * In the `submission_requirements` object, `name`, `rule (`pick` only)`, `count`, `from` properties MUST be supported.
 
+# OpenID for Verifiable Presentations over W3C Digital Credentials API
+
+The following requirements apply for both, the Wallet and the Verifier, unless specified otherwise:
```

Comment:
As above, please fix the grammar for _all_ bullet point lists in the document and consider removing "unless specified otherwise" (I don't immediately see where we do specify it otherwise)

Reply:
Ah, understood. Then I propose to find a wording in the bullet points to call out explicitly for each point who needs to follow it.

Re the editorial stuff - fine for me.